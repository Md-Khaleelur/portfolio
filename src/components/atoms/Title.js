import React, { Component } from 'react';
import "../template/styles.css";

export default class Title extends Component {
    render() {
        return (
            <div>
                <h2>{this.props.title}</h2>
            </div>
        )
    }
}
