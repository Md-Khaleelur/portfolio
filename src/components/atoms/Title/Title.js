import React, { Component } from 'react';

export default class Title extends Component {
    render() {
        return (
            <div           
              data-testid="title"
            >
                <h2>{this.props.title}</h2>
            </div>
        )
    }
}
