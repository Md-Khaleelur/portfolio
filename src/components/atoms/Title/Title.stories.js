import React from 'react';
import Title from './Title';

export default { 
    title: 'Title',

};

export const withText = () =><Title title="Mohammed Khaleelur Rehman"/>