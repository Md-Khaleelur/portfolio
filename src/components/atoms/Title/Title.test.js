import React from 'react';
import { create } from "react-test-renderer";
import { render } from '@testing-library/react';
import '@testing-library/jest-dom/extend-expect';
import Title from './Title';

describe("Title component",() =>{
    test("Matches the snapshot",() => {
        const title = create(<Title/>);
        expect(title.toJSON()).toMatchSnapshot();

    })
    test("Title is rendered",() => {
        const { getByTestId, getByText } = render(<Title/>);
        expect(getByTestId("title")).toBeInTheDocument();
        
    });
   

});