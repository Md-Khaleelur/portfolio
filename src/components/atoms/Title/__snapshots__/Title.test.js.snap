// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Title component Matches the snapshot 1`] = `
<div
  data-testid="title"
>
  <h2 />
</div>
`;
